src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -277,4 +279,87 @@ private static boolean continueConsuming(final int messagesConsumed, final int m
         30000,
         "Expected values not found in WindowStore"); }
 
+  static <K, V> Map<K, V> drainTableOutput(final String outputTopic,
+                                           final TopologyTestDriver topologyTestDriver,
+                                           final Deserializer<K> keyDeserializer,
+                                           final Deserializer<V> valueDeserializer) {
+
+    final Map<K, V> results = new LinkedHashMap<>();
```

Comment:
Why `LinkedHashMap` ?

Reply:
To improve readability. This way, when you print out the results (e.g., while writing a test, or playing around with the code, or looking at test failure output), the string form of the results is:
* stable order
* order depends on history of the output topic, so you can reason about what you expect to see